conf/machine/raspberrypi4-64.conf

```diff
@@ -19,10 +19,14 @@ RPI_KERNEL_DEVICETREE = " \
 SDIMG_KERNELIMAGE ?= "kernel8.img"
 SERIAL_CONSOLES ?= "115200;ttyS0"
 
-MACHINE_FEATURES_append = " armstub vc4graphics"
-VC4DTBO ?= "vc4-fkms-v3d"
```

Comment:
Dont we need `fkms` still for 64bit

Reply:
We do. That slipped in by mistake. Fixed.